### ISSUE TITLE
Discounts not displayed properly in Orders panel

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2773]
### Expected behavior
Discounts should show as a line item and be reflected in totals

### Actual Behavior
"Add Discount" is displayed but the actual discount is not

### Steps to Reproduce the Behavior
1. Login as an admin and set up shipping and payment methods
1. Add a discount code
1. Check out and use that code
1. Go to orders panel
1. Observe that discounts are not displayed

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.0-ce
```

**Comment by spencern** [comment]
May be related to #2763

**Comment by aaronjudd** [comment]
@zenweasel self-assigned and this hasn't been updated in 14 days. I'm assuming that this is a blocker as it probably affecting a single shop instance as well?  If it is working then we can at least remove the "marketplace" label.

**Comment by zenweasel** [comment]
As far as I remember (and as listed in the test steps) this affects single shop instances as well. I thought I was going to knock this out with the other order panel changes but apparently not. Unassigning myself for now and will grab it again when I have cleared the other stuff from my plate.

**Comment by zenweasel** [comment]
Tested again and verified that this happens in a single-shop instance. Discounts work fine just not displayed in the dashboard. Might just be a UI thing that got lost.

**Comment by zenweasel** [comment]
Closed via #2980
